go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -292,6 +296,8 @@ func Init() {
 type TabletConfig struct {
 	DB *dbconfigs.DBConfigs `json:"db,omitempty"`
 
+	UnmanagedTablet bool `json:"-"`
```

Comment:
`json:"-"` means we don't want this field to be printed. I think it should be `json:"unmanaged,omitempty"` instead.

Reply:
done.